ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-nat.j2

```diff
@@ -133,21 +133,36 @@ NAT profile VRF is: {{ profile.vrf }}
 {%         for pool in ip_nat.pools %}
 
 #### Pool: {{ pool.name }}
-
+{%             set pool_type = pool.type | arista.avd.default('ip-port') %}
 | Setting | Value |
 | -------- | ----- |
+| Type | {{ pool_type }} |
+{%             if pool_type == 'ip-port' %}
+{%                 if pool.prefix_length is arista.avd.defined %}
 | Pool Prefix Length | {{ pool.prefix_length }} |
-{%             if pool.utilization_log_threshold is arista.avd.defined %}
+{%                 endif %}
+{%                 if pool.utilization_log_threshold is arista.avd.defined %}
 | Pool Utilization Threshold | {{ pool.utilization_log_threshold }} % (action: log) |
-{%             endif %}
-
+{%                 endif %}
```

Comment:
Have you checked if this threshold could also work for `port-only` (I don't know)?

Reply:
Yes, I checked `port-only` has just `port` as option `utilization` is not part of it.